core/src/test/resources/config/customApplication.conf

```diff
@@ -0,0 +1,3 @@
+datastax-java-driver {
+  basic.request.timeout = 5 seconds
+}
```

Comment:
I think that we should add a property that has exactly the same key in each of those files(but different value) and write a test that validates which value is the final one (and which was overridden)

Reply:
I'm not sure if Typesafe Config formally defines an order of precedence. It's very unlikely that someone would use multiple formats anyway. Maybe I should even split the test into 3 ones using different names, but it's just convenient to do it all in one test.